images/scripts/lib/anaconda-comps.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm fairly sure this is copied from somewhere. Like that it's hard to maintain, you don't know the original source nor how it was modified, nor how to keep it up to date with future changes. Preferably replace this with some `curl` call to download it from some git, or at least add a comment with an URL?